playbooks/roles/deploy-acid/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# List of additional files to copy to acid/ dir on server
+# Use to e.g. deploy settings.yml, public key for ZuulManager etc.
+files_to_copy: []
```

Comment:
If it's not used, don't put it here as extra optional super hiper auto feature.
For now this list is empty which means unused.
Ansible is hard to maintain and hard to extend. Less is more.